src/test/ui/symbol-names/impl1.rs

```diff
@@ -3,6 +3,8 @@
 // revisions: legacy v0
 //[legacy]compile-flags: -Z symbol-mangling-version=legacy
     //[v0]compile-flags: -Z symbol-mangling-version=v0
+//[legacy]normalize-stderr-32bit: "hdb62078998ce7ea8" -> "SYMBOL_HASH"
+//[legacy]normalize-stderr-64bit: "h62e540f14f879d56" -> "SYMBOL_HASH"
```

Comment:
I don't think we care about hashes, you could probably use regex to always normalize away `h[0-9a-f]{16}`.

Reply:
(this scared me when I first saw it, but these are for the target, not the host, and symbol names don't have to be constant between targets)